prow/cmd/crier/README.md

```diff
@@ -40,7 +40,7 @@ You can check the reported result by [list the pubsub topic](https://cloud.googl
 
 ### [GitHub reporter](/prow/crier/reporters/github)
 
-You can enable github reporter in crier by specifying `--github-workers=1` flag. (We only support single worker for github, due to [#13306](https://github.com/kubernetes/test-infra/issues/13306))
+You can enable github reporter in crier by specifying `--github-workers=N` flag.
```

Comment:
Please clarify that `N` is a positive integer otherwise this will be taken literally. 😄